spec/integration/generators/blacklight/solr5_generator_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+require 'generators/blacklight/solr5_generator'
+
+describe Blacklight::Solr5Generator do
```

Comment:
Do we need to test these? It seems like we're testing that the Rails generator works.